fmfexporter/fmf_adapter.py

```diff
@@ -86,17 +86,17 @@ def convert_from(self, fmf_testcase: FMFTestCase):
         raise NotImplementedError()
 
     @abc.abstractmethod
-    def submit_testcase(self, fmf_testcase: FMFTestCase):
+    def submit_testcase(self, fmf_testcase: FMFTestCase, parse_response=False):
```

Comment:
I was thinking about it and I believe that parse_response should not be added here to the interface method. Instead this should be added as an extra argument to the polarion adapter itself (inside `polarion_args_parser.py`), keeping default as false.

Reply:
Done.

Reply:
Just make sure to add an extra argument to the polarion adapter or this will never be used.